src/main/webapp/resources/js/pages/analysis/components/AnalysisReferenceFileRenderer.jsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * The following import statements makes available all the elements
+ * required by the component
+ */
+
+import React, { useContext } from "react";
+import { getI18N } from "../../../utilities/i18n-utilties";
+import { AnalysisSamplesContext } from "../../../contexts/AnalysisSamplesContext";
+import { SPACE_LG } from "../../../styles/spacing";
+import { FONT_WEIGHT_DEFAULT } from "../../../styles/fonts";
+import { Row, Col, Icon, Button } from "antd";
+
+export function AnalysisReferenceFileRenderer() {
+  /*
+   * The following const statement
+   * make the required context which contains
+   * the state and methods available to the component
+   */
+  const { analysisSamplesContext } = useContext(AnalysisSamplesContext);
+
+  const renderReferenceFile = () => {
+    const referenceFile = [];
+
+    if (analysisSamplesContext.referenceFile.length === 0) {
+      return null;
+    } else {
+      referenceFile.push(
+        <div style={{ marginBottom: SPACE_LG }} key="samplesDiv-1">
+          <h4
```

Comment:
`<Title level={4}` or whatever it is.

Reply:
Fixed in 2804c23baa10fdf170ebd8b07a191142d0927b12